pennylane/circuit_drawer/utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains some useful utility functions for circuit drawing.
+"""
+
+
+def default_wire_map(ops):
+    """Create a dictionary mapping used wire labels to non-negative integers
+
+    Args:
+        ops Iterable[Operation]
+
+    Returns:
+        dict: map from wires to sequential positive integers
+    """
+
+    # Use dictionary to preserve ordering, sets break order
+    used_wires = {wire: None for op in ops for wire in op.wires}
+    return dict(zip(used_wires, range(len(used_wires))))
```

Comment:
Just double-checking: could a simpler version using `enumerate be used here?

Reply:
```suggestion
    used_wires = {wire: None for op in ops for wire in op.wires}
    return {wire: ind for ind, wire in enumerate(used_wires)}
```